minimal-go-app/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.16 AS build
+WORKDIR /go/src/github.com/PhilomathesInc/demo-applications/minimal-go-app
+COPY go.mod .
+COPY go.sum .
+COPY *.go .
+RUN CGO_ENABLED=0 GOOS=linux go build -o /minimal-go-app
+
+FROM gcr.io/distroless/base-debian10
```

Comment:
Is there a specific need for using this image?

Reply:
We can use `scratch` image to reduce the image size completely. But `alpine` or these _distroless_ images contain things like CA certificates which need to be added to a `scratch`. So overall `alpine` or _distroless_ will have better security features than `scratch` image.

Reply:
@mugdha-adhav We can use `scratch` image to reduce the image size completely. But `alpine` or these _distroless_ images contain things like CA certificates which need to be added to a `scratch`. So overall `alpine` or _distroless_ will have better security features than `scratch` image.